feature/experimental/policy/policy_vrf_selection/otg_tests/protocol_dscp_rules_for_vrf_selection_test/protocol_dscp_rules_for_vrf_selection_test.go

```diff
@@ -350,6 +384,7 @@ func getPBRPolicyForwarding(args *testArgs, rules ...*oc.NetworkInstance_PolicyF
 
 func TestPBR(t *testing.T) {
 	t.Logf("Description: Test RT3.2 with multiple DSCP, IPinIP protocol rule based VRF selection")
+	ondatra.DUT(t, "dut")
```

Comment:
Can we remove this line? Seems like it doesn't do anything.

Reply:
Removed